src/Core/System/Snippet/SnippetFixer.php

```
@@ -15,7 +18,19 @@ public function __construct(private readonly SnippetFileHandler $snippetFileHand
     {
     }
 
+    /**
+     * @deprecated tag:v6.8.0 reason:new-optional-parameter - Will get a second parameter `$invalidPluralization`
+     */
     public function fix(MissingSnippetCollection $missingSnippetCollection): void
+    {
+        /** @var Snippets $invalidPluralization */
+        $invalidPluralization = \func_num_args() === 2 ? func_get_arg(1) : [];
```

please surround that with a feature isActive condition so that this not used anymore, if 6.8.0.0 flag is active

---

additionally we should throw a deprecation message, if the second parameter is not provided

---

That feels quite unhandy, but I hope I get your concern.

---

with that we would make sure, it is correctly adjusted with 6.8.0.0

---

Hope its fine the way it is now 🙂